### PR TITLE
[IMP] product_cost_price_avco_sync: Fix test. Aleatory error

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa

### CODE REVIEW

**Comment by sergio-teruel** [comment]
Travis fails due to stock_return_request module..

File: product_cost_price_avco_sync/tests/test_product_cost_price_avco_sync.py [hunk 1]
```diff
@@ -59,24 +60,35 @@ def test_sync_cost_price(self):
...
         move_in_2.quantity_done = move_in_2.product_uom_qty
         picking_in_2.action_done()
         move_in_2.date = '2019-10-02 00:00:00'
+        sleep(1)
 
         move_out = self.picking_out.move_lines[:1]
         move_out.quantity_done = move_out.product_uom_qty
         self.picking_out.action_done()
         move_out.date = '2019-10-03 00:00:00'
+        sleep(1)
```
Comment (carlosdauden) [inline product_cost_price_avco_sync/tests/test_product_cost_price_avco_sync.py]: I think that sleep after mover out is not needed

File: product_cost_price_avco_sync/tests/test_product_cost_price_avco_sync.py [hunk 2]
```diff
@@ -59,24 +60,35 @@ def test_sync_cost_price(self):
...
 
         picking_out_2 = self.picking_out.copy()
         move_out_2 = picking_out_2.move_lines[:1]
         move_out_2.quantity_done = move_out_2.product_uom_qty
         picking_out_2.action_done()
         move_out_2.date = '2019-10-04 00:00:00'
+        sleep(1)
```
Comment (carlosdauden) [inline product_cost_price_avco_sync/tests/test_product_cost_price_avco_sync.py]: Same that above

**Review by carlosdauden — APPROVED** [review]

**Comment by carlosdauden** [comment]
Travis green

**Comment by pedrobaeza** [comment]
Please remove extra non needed sleep and I'll merge.

**Comment by sergio-teruel** [comment]
@carlosdauden  Changes done!!!

**Pull request merged**